Frameworks/Foundation/NSDictionary.mm

```diff
@@ -716,6 +716,20 @@ - (NSArray*)keysSortedByValueUsingComparator:(NSComparator)comparator {
 /**
  @Status Interoperable
 */
+- (NSArray*)keysSortedByValueWithOptions:(NSSortOptions)opts usingComparator:(NSComparator)cmptr {
+    NSMutableArray* ret = [NSMutableArray arrayWithArray:[self allKeys]];
```

Comment:
you've got to release it now!

---
In reply to: [71627991](https://github.com/Microsoft/WinObjC/pull/673#discussion_r71627991) [](ancestors = 71627991)